CDCgov/NEDSS-Modernization
# CNFT2-1421-last-numeric-link

## Description

When advancing to the last numeric link available in the pagination area, the application returns the modal frame and buttons with no data.

## Tickets

* [CNFT2-1421](https://cdc-nbs.atlassian.net/browse/CNFT2-1421)

## Steps to verify

1. Navigate to /page-builder/manage/pages, then click create new page
2. Once an event type is chosen, click the 'Search and add condition(s)' link to view the conditions
3. Navigate to all pages to make sure all pages have conditions and the last page is not blank


[CNFT2-1421]: https://cdc-nbs.atlassian.net/browse/CNFT2-1421?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
